portfolio/src/main/webapp/script.js

```diff
@@ -137,14 +166,16 @@ document.addEventListener('DOMContentLoaded', (event) => {
     const url = '/data?name=' + name + '&category=' + category + '&content=' + recommendation +
                '&comment=' + comment;
 
+
     fetch(scriptURL, { method: 'POST', body: new FormData(form), mode: 'no-cors'})
-    .catch(error => respond('Uh oh, error: ' + error.message))
     .then(() => {
-        fetch (url, {method: 'POST'});
-    }).then(() => {
-        getData();
-        respond("Thank you for your recommendation :)");
+      fetch (url, {method: 'POST'});
     })
+    .then(() => {
+      getData();
+      respond("Thank you for your recommendation :)");
+    })
+    .catch(error => respond('Uh oh, error: ' + error.message))
```

Comment:
Does this need surrounding parentheses so that it has access to the respond method?
`catch((error) => respond(...)`

Reply:
oh i think if there's only one parameter it should work? but I never tested it. I added the parenthesis anyway!

Reply:
Oh yeah you're right. While I was looking this up, I found in the [eslint guidelines](https://eslint.org/docs/rules/arrow-parens) that they suggest always using parentheses to eliminate ambiguity. I think you don't need to follow that here since you aren't using that linter, but just an FYI!

Reply:
ah okay, will remember!